context/http.go

```diff
@@ -175,6 +175,19 @@ func GetResponseLogger(ctx Context) Logger {
 	return l
 }
 
+func GetErrorResponseLogger(ctx Context, errorMap map[string]interface{}) Logger {
```

Comment:
`GetResponseLogger` only knows about the context interface.  `Errors` is a member of the context struct defined in the handlers package, so it cannot look them up.